umh.docs.umh.app/content/en/docs/getstarted/movingtoproduction.md

```diff
@@ -26,124 +20,124 @@ weight = 5000
-
-5. Chose the **VM** or **Edge Deice** panel, depending on your choice and 
-   click on **continue** again. Follow the instruction to set up your instance.
-   
-6. When you are done and the instance is running, remember to place it in
-   the same network as your machines. The same applies for the device
-   that is running the Management Console. 
-
-7. Click on the tile of the new instance inside the overview
-   and open the Node-RED popup. Use this
-   [list](https://umh.docs.umh.app/docs/features/data-connectivity-node-red/#what-can-i-do-with-it)
-   to learn how to connect your machines to Node-RED.
-   If the provided help is not sufficient, you can ask for help on our Discord.
-
-8. Use Node-RED to generate a new dataflow by using your machines´ data as input.
-   You can use this collected data in other tools, for example to build a
-   dashboard in Grafana. 
+  Manufacturing Hub and learn how to use them.
+- [Architecture](/docs/architecture/) to learn what is behind the United
+  Manufacturing Hub and how everything works together.
+
+Ready to transition to production? Continue reading to discover how to install UMH and seamlessly connect multiple machines to your instance.
+
+## Set up your first instance and connect to a few machines
+
+If you want to get a first impression of the UMH in a production environment, connecting to machines on your shop floor, follow these steps:
+
+#### 1. Choose an Edge Device or VM (Recommended: VM)
+
+Before starting the installation process, decide whether to use virtual machine (VM), or a generic server or edge device. For ease of setup, we recommend using a VM. Ensure that the selected device has network access to the machines.
+
+#### 2. Select Machines with OPC UA for Testing
+
+For testing purposes, it's recommended to use machines with OPC UA. If your machines use other protocols, consider Node-RED as an alternative for data connection. Check the [list of supported protocols](https://umh.docs.umh.app/docs/features/data-connectivity-node-red/) and how to connect them to Node-RED.
+
+#### 3. Installation Process
+
+The installation is well documented in the [first chapter](/docs/getstarted/installation/#installation-steps). But here's a quick overview:
+
+- Click on the **+ Add Instance** button in the instance dashboard of the Management Console, redirecting you to the installation process page.
+
+- Select the **Install UMH Only** option, redirecting you to the install command generation page.
 
+- Finally, follow the provided instructions to set up your instance. If everything went well, there should be a button at the bottom right corner
+  of the page, redirecting you back to the instance dashboard.
+
+#### 4. Network Configuration
+
+Once your UMH instance is up and running, ensure it is placed in the same network as your machines. Additionally, ensure that the device running the Management Console is also within the same network.
```

Comment:
Maybe give some background information why it is crucial, e.g., it is not required for setting up managing and monitoring, but if you don't have a VPN you might not be able to Grafana or Node-REd (or something similar)